website/docs/r/ddosbgp_instance.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_ddosbgp_instance"
+sidebar_current: "docs-alicloud-resource-ddosbgp-instance"
+description: |-
+  Provides a Alicloud 
+Anti-DDoS Advanced(Ddosbgp) Instance Resource.
+---
+
+# alicloud_ddoscoo_instance
+
+Anti-DDoS Advanced instance resource. "Ddosbgp" is the short term of this product.
+
+-> **NOTE:** The endpoint of bssopenapi used only support "business.aliyuncs.com" at present.
+
+-> **NOTE:** Available in 1.56.0+ .
+
+## Example Usage
+
+Basic Usage
+
+```
+provider "alicloud" {
+  endpoints {
+    bssopenapi = "business.aliyuncs.com"
+  }
+}
+
+resource "alicloud_ddosbgp_instance" "instance" {
+  name              = "yourDdosbgpInstanceName"
+  base_bandwidth    = "20"
+  bandwidth         = "101"
+  ip_count          = "100"
+  ip_type           = "1"
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `type` - (Required,ForceNew) Type of the instance. Valid values: 1,0.
```

Comment:
please show more details for `0` and `1`, like `0` for professional and `1` for enterprise